etc/mpd.profile

```diff
@@ -0,0 +1,34 @@
+# Firejail profile for mpd
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/mpd.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+
+noblacklist ${HOME}/.mpdconf
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+
+caps.drop all
+netfilter
+no3d
+nodvd
+nogroups
```

Comment:
mpd should be member of `audio` group, contradicting `nogroups`. Also `netfilter` looks problematic to me.